pkg/config/config.go

```diff
@@ -133,6 +134,13 @@ func initializeFlags() *EnvironmentFlags {
 	flag.StringVar(&f.TknVersion, "tknversion", defaultTkn,
 		"Provide tknversion to download specified cli binary you'd like to use for these tests. By default `0.6.0` will be used.")
 
+	defaultClusterArch := os.Getenv("ARCH")
```

Comment:
Are we setting this in the testing task?

Reply:
Not yet, I updated description of this PR. I will do it after this is merged.